packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -61,14 +61,8 @@ export default class Toolbar extends React.Component<Props> {
 
     renderContent(content: React.Node) {
         const contentArray = Array.isArray(content) ? content : [content];
-        // Remove empty/null/undefined elements, then map to Views
-        return contentArray.filter((c) => c).map((content, i) => (
-            <View
-                style={[sharedStyles.content, sharedStyles.verticalAlign]}
-                key={i.toString()}
-            >
-                {content}
-            </View>
+        return contentArray.map((content, i) => (
+            <View key={i.toString()}>{content}</View>
```

Comment:
Requesting one last change: this can be
```
renderContent(content: React.Node) {
    return Array.isArray(content) ? content : [content];
}
```

Reply:
Yeah, agreed with Kevin - I don't think we want the <View> wrapper here as it means that the Flex styling of the individual content items won't come through.

Reply:
Ok! So thinking about this problem more - it looks like you want to avoid having to add keys to the items that you're inserting. I think the easiest way to do this is to pass in a React.Fragment for the value of leftContent/rightContent instead of an array.

In the example below you currently have:

```
<Toolbar size="small" rightContent={[smallButton, strut, secondaryButton]} />
```

We can instead do:

```
<Toolbar size="small" rightContent={<React.Fragment>
	{smallButton}
	{strut}
	{secondaryButton}
</React.Fragment>} />
```

Once we finish upgrading to Babel 7 the `<React.Fragment>`s can just become: `<>...</>`.

So in this case I think we can just get rid of "renderContent" entirely and just put the contents directly into the page, don't bother with the key insertion at all (leave that up to the user - if they actually want an array then it's up to them to have keys on the items).

Reply:
Given what I said above I think we should remove the default values of leftContent and rightContent above (or make them `null` and make the properties required in the Flow type).